### PR TITLE
Updated status cake alerts goup list

### PR DESCRIPTION
**Pull request by VenkataChalla554**
### Context

Added contact group list for status cake alert.

### Changes proposed in this pull request

Added contact group list for status cake alert.

### Guidance to review

Include any useful information needed to review this change.
Include any dependencies that are required for this change.

### Checklist

-   [x] Attach to Trello card
-   [x] Rebased master
-   [x] Cleaned commit history
-   [ ] Tested by running locally
-   [x] Reminder created to manually clean any removed app settings post deployment

### CODE REVIEW

**Comment by VenkataChalla554** [comment]
[Review StatusCake contact groups](https://trello.com/c/Z6eliKhq/1654-review-statuscake-contact-groups)

File: terraform/variables.tf [hunk 1]
```diff
@@ -142,9 +142,9 @@ variable "domain" {
 }
 
 variable "statuscake_ssl_contact_group" {
-  type        = string
-  default     = null
-  description = "ID of the StatusCake contact group. If empty, SSL check is not enabled"
+  type        = list(string)
+  default     = []
+  description = "ID's of the StatusCake contact group. If empty, SSL check is not enabled"
```
Comment (saliceti) [inline terraform/variables.tf:147]: Typo: IDs

File: terraform/variables.tf [hunk 2]
```diff
@@ -142,9 +142,9 @@ variable "domain" {
 }
 
 variable "statuscake_ssl_contact_group" {
```
Comment (saliceti) [inline terraform/variables.tf:144]: Can it be renamed to statuscake_ssl_contact_groups? as it contains multiple groups

Comment (VenkataChalla554) [reply terraform/variables.tf]: Fixed this .

**Review by saliceti — APPROVED** [review]

**Pull request merged**